viz_scripts/mode_purpose_share.ipynb

```diff
@@ -25,166 +25,146 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "year = 2020\n",
-    "month = 11\n",
-    "program = \"prepilot\""
+    "year = 2021\n",
+    "month = 9\n",
+    "program = \"prepilot\"\n",
+    "study_type = \"program\"\n",
+    "mode_of_interest = \"Pilot ebike\""
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "respiratory-gibraltar",
+   "id": "traditional-queens",
    "metadata": {},
    "outputs": [],
    "source": [
-    "import scaffolding"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "upset-transition",
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "import pandas as pd"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "unauthorized-miami",
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "tq = scaffolding.get_time_query(year, month)"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "authorized-living",
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "participant_ct_df = scaffolding.load_all_participant_trips(program, tq)"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "checked-thumb",
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "labeled_ct = scaffolding.filter_labeled_trips(participant_ct_df)"
+    "from collections import defaultdict\n",
+    "import datetime\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import seaborn as sns\n",
+    "\n",
```

Comment:
we can remove this notebook since all the examples here are better covered in `generic_metrics`. Using this as the baseline example will just lead to bad examples (e.g. having the change the file name every time).

Reply:
Removed this notebook